docs/quickstart.md

```diff
@@ -0,0 +1,13 @@
+---
+id: quickstart 
+title: Quick Start
+sidebar_label: Quick Start
+---
+
+This section is a quick guide on how to do most things on Avolites Titan
+consoles. Each section has a link to take you to the appropriate manual
+section for more details.
+
+Softkeys (the A-G menu buttons next to the screen) are shown like 
+\[This\]. Physical labelled buttons on the console are
+shown like \<This\>.
```

Comment:
This could be places in the introduction, or the quick start and introduction could be merged so there is less to scroll through on the console.

Reply:
It would be a good idea to put a section on the introduction page giving the conventions of the manual including this. There was a page like this in the old pdf manual. Will add a section.

I see "Introduction" and "Quick Start" as different things and think they should be kept separate. "Introduction" is for everyone including the beginner, but "Quick Start" is just for the the experienced MA or Chamsys programmer who needs a quick reference on how to do something on Titan.